### DISCUSSION TITLE
Rusdesk Pro: Reset MFA for Admin account

### DISCUSSION THREAD

**Discussion by gentilsol** [rustdesk/rustdesk-server-pro#645] [Q&A] [answered]
My SMTP server has changed.
As a result, no e-mail is now sent for the MFA. This means I can no longer access the admin interface.

Is there a way to deactivate or reset the MFA?

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk-server-pro/issues/99#issuecomment-1747965887

**Reply by gentilsol** [reply]
thanke you for the fast respons.
But this not work with an docker-compose

**Reply by rustdesk** [reply]
```
volumes:
      - ./data:/root
```

**Reply by gentilsol** [reply]
i don't understand right now, i'm still new to the subject

**Reply by rustdesk** [reply, marked as answer]
Though you are using docker-compose, you can still find sqlite3 db files on your local directory. 

Below part in the `docker-compose.yml` file tells how does docker map volume in container to your local directory, https://stackoverflow.com/questions/40905761/how-do-i-mount-a-host-directory-as-a-volume-in-docker-compose.
```
volumes:
      - ./data:/root
 ```

Once you find the sqlite3 db files, you can modify it following this link, https://github.com/rustdesk/rustdesk-server-pro/issues/99#issuecomment-1747965887, supposing you know sqlite3 and some knowledge about sql.

**Reply by gentilsol** [reply]
Thank you really mutch
